selendroid-server/src/test/java/io/selendroid/server/BaseTest.java

```diff
@@ -45,12 +45,13 @@
   public static final String host = "127.0.0.1";
   protected AndroidServer server;
   public static final String ANY_STRING = "ANY-STRING";
+  private static String automationName = "selendroid";
```

Comment:
This should be final and in caps.

`private static final String AUTOMATION_NAME = "selendroid";`